community/py-flask-babel/APKBUILD

```diff
@@ -0,0 +1,71 @@
+# Contributor: Eivind Uggedal <eu@eju.no>
+# Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
+pkgname=py-flask-babel
+_pkgname=Flask-Babel
+pkgver=0.12.2
+pkgrel=0
+pkgdesc="Flask i18n/l10n support"
+url="https://pypi.python.org/pypi/Flask-Babel"
+arch="noarch"
+license="BSD"
+makedepends="
+	py-setuptools
+	py2-flask
+	py3-flask
+	python2-dev
```

Comment:
for noarch packages python2-dev and python3-dev are most of the time not required, try adding py3-setuptools instead.